src/plans/record.jl

```diff
@@ -15,6 +15,7 @@ called from within [`stop_solver!`](@ref) which returns true afterwards.
 * `recorded_values` an `Array` of the recorded values.
 """
 abstract type RecordAction <: AbstractStateAction end
+status_summary(ra::RecordAction) = "$(ra)"
```

Comment:
This is the same as default.

Reply:
Is already removed in the next commit – I introduced the default a bit late. I am currently working on record code cov anyways.